Extend/Model/Order.php

```diff
@@ -30,7 +30,7 @@ class Order extends Order_parent
     const STATE_PENDING = 'pending';
     const STATE_AUTHORIZED = 'authorized';
     const STATE_PROCESSING = 'processing';
-    const STATE_CANCELED = 'canceled';
+    const STATE_CANCELLED = 'cancelled';
```

Comment:
please use all states from paymentSDK with the new release they are all there

Reply:
👍